styles.css

```diff
@@ -1,3 +1,150 @@
+@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@600&display=swap");
```

Comment:
nothing wrong with this, but i think it's easier to keep track of fonts if they're all downloaded in the same place - so either download all of them in a css file or all of them in the `head` of the HTML. 

i prefer the latter solution because in React that'll increase visibility